src/main/kotlin/game/Game.kt

```diff
@@ -588,28 +590,30 @@ internal fun sendPlayerInfo(
                                                     roleDesc
                                                 )*/
             try {
-                val chat = ChatId.fromId(con.playerId)
-                bot.sendMessage(
-                    chat,
-                    "Ведущий начал игру",
-                    replyMarkup = mafiaKeyboard(chatId)
+                var msgId = sendMessage(
+                    bot,
+                    chatId,
+                    "Ведущий начал игру"
                 )
-                val res = bot.sendMessage(
-                    chat,
-                    getRoleDesc(role),
-                    parseMode = ParseMode.HTML,
+                updateMessage(
+                    bot, chatId, msgId,
+                    inlineKeyboard { mafiaKeyboard(chatId) }
                 )
-                if (res.isSuccess) {
-                    val msgId = res.get().messageId
-                    bot.editMessageReplyMarkup(
-                        chat,
-                        msgId,
-                        replyMarkup = inlineKeyboard {
+                msgId = sendMessage(
```

Comment:
Whole idea of this update was to allow for sending keyboard with message even if it needs a message id
Contrary happens here